tests/platform_tests/test_intf_fec.py

```diff
@@ -91,3 +94,32 @@ def test_config_fec_oper_mode(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
 
             if not (fec == "rs"):
                 pytest.fail("FEC status is not restored for interface {}".format(intf['interface']))
+
+
+def test_verify_fec_stats_counters(duthosts, enum_rand_one_per_hwsku_frontend_hostname,
+                                   enum_frontend_asic_index, conn_graph_facts):
+    """
+    @Summary: Verify the FEC stats counters are valid
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+
+    if not is_supported_platform(duthost):
+        return
+
+    logging.info("Get output of '{}'".format("show interfaces counters fec-stats"))
+    intf_status = duthost.show_and_parse("show interfaces counters fec-stats")
+
+    for intf in intf_status:
+        sfp_presence = duthost.show_and_parse("sudo sfpshow presence -p {}"
+                                              .format(intf['iface']))
+        if sfp_presence:
+            presence = sfp_presence[0].get('presence', '').lower()
+            if presence == "not present":
+                continue
```

Comment:
@vvolam if SFP is not present, FEC stats is not available or its showing 0?

Reply:
@prgeor If SFP is not present, counters are showing 0. I have removed the SFP present check in the latest, as it is not required. Thanks!